.travis.yml

```diff
@@ -24,11 +24,12 @@ before_script:
 
 env:
   global:
-    - PROXY_TESTS_DIR=files/default/scripts
+    - PROXY_TESTS_DIR=/tmp/proxy_tests
     - PROXY_TESTS_REPO=$PROXY_TESTS_DIR/repo
 
 script:
   - chef-client --version
+  - chef-client -z -o proxy_tests::render
```

Comment:
Can you add a comment here explaining why the berkshelf project needs to run this and other projects (like proxy_test, chef, Test Kitchen) don't?